package.json

```diff
@@ -92,7 +92,7 @@
   },
   "dependencies": {
     "@elastic/datemath": "link:packages/elastic-datemath",
-    "@elastic/elasticsearch": "npm:@elastic/elasticsearch-canary@^7.12.0-canary",
+    "@elastic/elasticsearch": "npm:@elastic/elasticsearch-canary@^7.12.0-canary.4",
```

Comment:
It looks like we missed several canary releases. @elastic/kibana-operations is it possible to automate the update when a new version is available?

Reply:
We have automation setup for 7.x and master, I don't think we want to be upgrading version branches after feature freeze without a specific need to, but maybe that's a bad assumption?

Reply:
@spalger agree, but `@elastic/elasticsearch` is a bit different. It can be released several times during FF. Is there a way to automate version bump for the ES client package only?